crates/sargon-uniffi/src/signing/subintent/sign_with_factors_outcome.rs

```diff
@@ -0,0 +1,58 @@
+use crate::prelude::*;
+use sargon::SubintentHash as InternalSubintentHash;
+
+type InternalSignWithFactorsOutcomeForSubintent =
+    sargon::SignWithFactorsOutcome<InternalSubintentHash>;
+
+#[derive(Clone, PartialEq, Eq, uniffi::Enum)]
```

Comment:
You should be able to use InternalConversion macro

Reply:
I dont think he can? the `Internal` uses generics, which prevents usage of `InternalConversion` I think?

Reply:
`InternalSignWithFactorsOutcomeForSubintent` is a concrete type, not generic